source/shared/components/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class App extends Component {
```

Comment:
For components that aren't pure functions, use [react-stamp](https://github.com/stampit-org/react-stamp).